src/clj/rems/api/services/licenses.clj

```diff
@@ -0,0 +1,79 @@
+(ns rems.api.services.licenses
```

Comment:
consider naming this `rems.services.licenses`

Reply:
Hmm, I find it slightly "non-informative" to have a directory structure where src/clj/rems/services, /api, and /db directories are all at the same level (together with, e.g., /css), while I feel that services is kind of subordinate to api -- I wouldn't understand from that directory structure alone what "services" is related to. Though I guess it could be rems.db.services, as well.

This is just from the directory structure viewpoint, but I've understood that the namespace naming needs to conform to that.

Reply:
Fair enough